plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -2692,6 +2692,10 @@ fc::variant chain_plugin::get_log_trx(const transaction& trx) const {
     }
     return pretty_output;
 }
+
+const controller::config* chain_plugin::chain_config() const {
```

Comment:
I think this should return a `const controller::config&` and `EOS_ASSERT` if null.